src/COVID19/model.py

```diff
@@ -178,7 +191,60 @@ def _get_base_param_from_enum(param):
             break
     return base_name, enum_val
 
+class VaccineSchedule(object):
+    def __init__(
+        self,
+        frac_0_9   = 0,
+        frac_10_19 = 0,
+        frac_20_29 = 0,
+        frac_30_39 = 0,
+        frac_40_49 = 0,
+        frac_50_59 = 0,
+        frac_60_69 = 0,
+        frac_70_79 = 0,
+        frac_80    = 0,
+        vaccine_type    = 0,
```

Comment:
Could we specify a type here for vaccine_type. I would think of type `Union[VaccineTypeEnum, int]` and then something in the `__init__` method which knows how to deal with each case. I guess `enums` could be converted to integer on the user side but if so, might be worth making this explicit so anyone using `VaccineTypeEnum`s will get picked up by linters. When creating `VaccineSchedule` objects I ran into an issue using `Model.vaccinate_schedule` because I had passed vaccine a variable of class `VaccineTypeEnum` and not a value. When `covid19.intervention_vaccinate_age_group` is called it expects this to be of type short and raises an exception.

The other alternative is to specify the third argument passed to `covid19.intervention_vaccinate_age_group` to be `schedule.vaccine_type.value` but then would presumably need to ensure `vaccine_type` is of class `VaccineTypeEnum` first.